src/api/argo-templates/_test_data_/worker.rebuild.yaml

```diff
@@ -92,89 +92,42 @@ spec:
     - name: main
       dag:
         tasks:
-          - name: install-csi
-            templateRef:
-              name: ssh-template
-              template: shell-script
+          - name: before-all
```

Comment:
This is just a preference thing and not necessary to change. "Before-all" could be renamed to something like "prepare for upgrade". In my opinion, this makes more sense from a user perspective looking at the workflow. This is the same with some of the other stages like "before-each"